AK/Checked.h

```diff
@@ -138,7 +138,8 @@ class Checked {
     template<typename U>
     constexpr Checked& operator=(U value)
     {
-        return *this = Checked(value);
+        *this = Checked(value);
+        return *this;
```

Comment:
Question: What's the difference?
(I'm just curious.)

Reply:
No assignment in return statement.

Reply:
Resolved (I can't mark it myself)